### DISCUSSION TITLE
Project size management tools

### DISCUSSION THREAD

**Discussion by erindelaney** [redivis/meta#155] [Feature requests]
Users need a way to better manage their storage resources while still keeping their projects accessible. 

Considerations:
- Best way to store a completed project that is no longer being updated, but needed for future reference.
- Way for user to reduce storage (and potentially visual overhead) on existing projects as they go.
- Tables in projects constitute project size. How to keep which tables are necessary and remove or reduce ones that aren't.

**Comment by imathews** [comment]
This is no longer relevant, since the size of tables in a project don't contribute to any storage quotas
